commands/weather/weather.js

```diff
@@ -6,7 +6,7 @@ const geocodeEndpoint = 'https://maps.googleapis.com/maps/api/geocode/json?key=g
 const darkskyEndpoint = 'https://api.darksky.net/forecast/key/lat,lng';
 
 function fahrenheitToCelcius(degree) {
-  return ((degree - 32) * 5 / 9).toFixed(0);
+  return (((degree - 32) * 5) / 9).toFixed(0);
```

Comment:
can we use http://eslint.org/docs/rules/no-mixed-operators#allowsameprecedence instead?